tests/type-tests/test/model.test-d.ts

```diff
@@ -0,0 +1,10 @@
+import {
+  ChatModel,
+  AzureOpenAiChatModel,
+  EmbeddingModel
+} from '@sap-ai-sdk/core';
+
+expect<ChatModel>('custom-model');
```

Comment:
I assume you also checked autocompletion works manually? 😊

Reply:
Sure see the screenshot above. Works like a charm.

Reply:
Oops, totally ignored the description. 🙊